db/schema.rb

```diff
@@ -19,8 +19,32 @@
     t.string "title"
   end
 
+  create_table "reviewings", force: :cascade do |t|
```

Comment:
There's not a many-to-many relationship between reviews and books (e.g. a review isn't for many books, it's only for one book).

The many-to-many is between users and books, and we're naming it `reviews`, and it's holding additional information (e.g. body, and rating).